cmd/context/update-kubeconfig.go

```diff
@@ -23,6 +23,17 @@ import (
 	oktetoLog "github.com/okteto/okteto/pkg/log"
 	"github.com/okteto/okteto/pkg/okteto"
 	"github.com/spf13/cobra"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+)
+
+var (
+	oktetoExecConfig = &clientcmdapi.ExecConfig{
```

Comment:
could we use the value directly [here](https://github.com/okteto/okteto/pull/3887/files#diff-e68e63e6d73f6931e3cc1904794fbb6f72948431573b0d73036c90e69f2022ffR109) instead declare a global variable?

Reply:
yes, i removed the global and added a function to include this with params